site/_posts/2021-12-07-Announcing-Morphy-Beta.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Announcing Morphy Beta
+author: Fryguy
+date: 2021-12-07 14:56:43 UTC
+comments: true
+published: true
+tags: releases announcements
+---
+
+The first beta release of ManageIQ Morphy is here!  We've been working hard and have a lot of new features you can see in the [roadmap](https://github.com/orgs/ManageIQ/projects/13#column-15656536). Here are the major highlights for this new release:
+
+New Providers!
+* [IBM PowerVC](https://www.ibm.com/products/powervc)
+* [Oracle Cloud](https://www.oracle.com/cloud)
+
+New Kubernetes Providers!
+* [Azure Kubernetes Service (AKS)](https://azure.microsoft.com/en-us/services/kubernetes-service)
+* [Google Kubernetes Engine (GKE)](https://cloud.google.com/kubernetes-engine)
+* [IBM Cloud Kubernetes Service (IKS)](https://www.ibm.com/cloud/kubernetes-service)
+* [Oracle Kubernetes Engine (OKE)](https://www.oracle.com/cloud-native/container-engine-kubernetes)
+
+New support for Cloud Databases
+* Amazon RDS
+* Azure Database for SQL Server, PostgreSQL, MySQL, and MariaDB
+* Google Cloud SQL
+* IBM Cloud Databases
+* Oracle Cloud Databases and MySQK Databases
+
+Provider Enhancements
+* IBM Cloud VPC
+  * Event Collection support
+  * Targeted Refresh for VMs, Templates, and Flavors
+  * Create/Delete of Networks, Subnets, and Volumes
+  * Collection of Cloud Volume Types
+  * Link Availability Zone to Cloud Subnets
+  * Link Floating IPs to Instances
+  * Provision VMs from Images/Templates
+  * Delete VMs
+* IBM Cloud PowerVS
+  * Targeted Refresh
+  * Reporting on virtual columns for Power related attributes
+  * Reporting templates for PowerVS/PowerVC
+* IBM PowerVC
+  * New dedicated provider!
+  * Collection of Cloud Volume Types
+  * Support software license fields during provisioning
+* IBM AutoSDE
+  * Predefined roles
+  * Edit Storage Device attributes
+  * Configure a Volume for Expansion
+  * Validation check for CHAP Username and Secret together
+  * Creation of multiple volumes in a single view/click
+  * Edit Volume Mappings
+  * Cross link between Cloud Volume and Host Initiator
+  * Creation of host initiator with multiple WWPN for FC port
+
+API
+* Throttle API requests
+* Allow filtering '=' via arrays
+
+Core
+* [Embedded Ansible] Configurable `verify_ssl` for git repositories
+* [Appliances] Move from CentOS 8 to CentOS 8 Stream
+* [Installation] Gracefully handle installation with non-superuser PostgreSQL account
```

Comment:
> more was done on the core side (@jrafanie @kbrock),

Core/UI: Exclude deprecated templates from provisioning dialogs since google retains no longer used templates, resulting in slow pages before this change.

Faster database migrations: https://github.com/ManageIQ/manageiq/pull/21259

Technically, the "gracefully handle installation with non-superuser PostgreSQL account" includes adding support to run both the publication and subscription databases on the same machine for testing purposes and scripts to help setup and verify this setup.

Reply:
Yeah I was leaving out the "developer" stuff because the audience for this is a user.  I like the faster DB migrations and performance changes though. Thanks!

Reply:
> Core/UI: Exclude deprecated templates from provisioning dialogs since google retains no longer used templates, resulting in slow pages before this change.

do you have a PR link for this one @jrafanie ?

Reply:
https://github.com/ManageIQ/manageiq/pull/21455